Source/ACE/Entity/Landblock.cs

```diff
@@ -87,8 +85,8 @@ public Landblock(LandblockId id)
             var creatures = DatabaseManager.World.GetCreaturesByLandblock(this.id.Landblock);
             foreach (var c in creatures)
             {
-                Creature cwo = new Creature(c);
-                worldObjects.Add(cwo.Guid, cwo);
+                // Creature cwo = new Creature(c);
+                // worldObjects.Add(cwo.Guid, cwo);

```

Comment:
Reverted it.  For some reason that section causes an Exception for me, so was easier to comment it out.